packages/docs/stories/components/option.stories.ts

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable arrow-body-style */
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable import/no-relative-packages */
+
+import '../../../components/src/components/option/option';
+import type { Meta, StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+import {
+  generateScreenshotStory,
+  generateStoryDescription,
+  storybookDefaults,
+  storybookHelpers,
+  storybookTemplate,
+} from '../../src/helpers/component.js';
+
+const { args: defaultArgs, argTypes } = storybookDefaults('syn-option');
+const { overrideArgs } = storybookHelpers('syn-option');
+const { generateTemplate } = storybookTemplate('syn-option');
+
+const meta: Meta = {
+  args: overrideArgs([
+    { name: 'default', type: 'slot', value: 'Option 1' },
+    { name: 'value', type: 'attribute', value: 'Option 1' },
+  ], defaultArgs),
+  argTypes,
+  component: 'syn-option',
+  parameters: {
+    docs: {
+      description: {
+        component: generateStoryDescription('option', 'default'),
+      },
+      story: {
+        height: '250px',
+      },
+    },
+  },
+  title: 'Components/syn-option',
+};
+export default meta;
+
+type Story = StoryObj;
+
+export const Default = {
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('option', 'default'),
+      },
+    },
+  },
+  render: (args: any) => html`
+    <syn-select label="Select one">
+      ${generateTemplate({ args })}
+      <syn-option value="2">Option 2</syn-option>
+    </syn-select>
+  `,
+} as Story;
+
+export const Disabled: Story = {
+  name: 'Disabled',
+  parameters: {
+    docs: {
+      description: {
+        story: generateStoryDescription('option', 'disabled'),
+      },
+    },
+  },
+  render: () => html`
+    <syn-select label="Select one" label="Select one">
```

Comment:
Redundant label prop
```suggestion
    <syn-select label="Select one">
```